package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@opengovsg/formsg-sdk",
-  "version": "0.12.0",
+  "version": "0.14.0-alpha.0",
```

Comment:
/nit because we're changing the function definition (from sync to async), I think this warrants a bump on the major  version to 1.0.0 (can declare that as 1.0.0-alpha.0 or 1.0.0-rc.0). 

We'll need a changelog with migration notes as well, this can be done later! 🙏

Reply:
definitely!
migration notes as in smth like `MIGRATION.md`, [form guide](https://guide.form.gov.sg/), or both?

Reply:
`MIGRATION.md` like [storybook](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) or better